binding/binding.go

```diff
@@ -0,0 +1,28 @@
+package binding
```

Comment:
why `binding` instead of `splitio-api-binding`? `binding` seems a little too generic

Reply:
yeah I agree, I named it `binding` because I thought that will be too long. GO convention suggests a package name to be short and clear, all lower cases, and with no under_scores or mixedCaps.
https://blog.golang.org/package-names

`splitioapibinding` or `splitioapibind` or `apibind`, will these be better?

Reply:
this isn't something we'd expect users of this module to call, right?  maybe `api` is a good short and simple option.

Reply:
yeah I think users will only interact with the `serializer`.
I'll change it to `api`, thanks for the suggestion!